components.yaml

```diff
@@ -70,6 +70,16 @@ GEOSchem_GridComp:
   tag: v1.5.0
   develop: develop
 
+HEMCO:
+  local: ./src/Components/@GEOSgcm_GridComp/GEOSagcm_GridComp/GEOSphysics_GridComp/@GEOSchem_GridComp/Shared/HEMCO/@HEMCO
+  remote: ../HEMCO.git
+  branch: GEOS/main
+
+GEOS-chem:
```

Comment:
You might want to call this `geos-chem` so that it matched the repo you checkout into `@geos-chem`. Not necessary as all it is is the "name" mepo uses internally for it so when you run `mepo command repo` it uses this key for the repo.

Reply:
I'm not very knowledgeable about sub-repo's, but why is GEOSCHEMchem_GridComp/@geos-chem  preferred to just  @GEOSCHEMchem_GridComp  ?  In other words, do we really need two directory levels when one was sufficient before?

Reply:
I imagine it will become clear to me when the CHEM PR is available, and I can see what's left under the first level.